demos/motion_tracking/main_motion_tracking.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+
+from threading import Thread
+from functools import partial
+from bokeh.plotting import figure, curdoc
+from bokeh.layouts import gridplot, Spacer
+from bokeh.models import LinearColorMapper, ColorBar, Title, Button
+from bokeh.models.ranges import DataRange1d
+import sys
+from multiprocessing import Pipe
+from motion_tracking_network import MotionNetwork
```

Comment:
How about renaming the `motion_tracking_network` module to `motion_tracker` and `MotionNetwork` to `MotionTracker`?

Reply:
Done